### PR TITLE
Added configurable margin which is applied to touch area

### PR DESCRIPTION
**Pull request by ppodhajski**

### CODE REVIEW

**Comment by ppodhajski** [comment]
retest this please

**Comment by ppodhajski** [comment]
retest this please

**Comment by ppodhajski** [comment]
retest this please

File: doc/Changelog.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ Changelog {#changelog}
 
 * [298](https://github.com/BlueBrain/Tide/pull/298):
   New option: start/resume movie playback when window is maximized.
+* [300](https://github.com/BlueBrain/Tide/pull/300):
```
Comment (tribal-tec) [inline doc/Changelog.md]: 303

**Pull request merged**